### PR TITLE
Provide some baseline intallation instructions; it's not as simple as…

### PR DESCRIPTION
**Pull request by sacrebleu**
… bundle install and run ;)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,4 +2,38 @@
...
+
+##### Ruby
+
+Using [RVM](https://rvm.io/rubies/installing)
+
+    $ rvm install 2.5.0
+    
+Using [RBEnv](https://github.com/rbenv/rbenv#installing-ruby-versions)
+
+    $ rbenv install 2.5.0
```
Comment (WaKeMaTTa) [inline README.md]: Can you move `rbenv` case above of `rvm` case. Because `rbenv` easyer than `rvm`. Thank you.

Comment (sacrebleu) [reply README.md]: 👍 Done

**Comment by colszowka** [comment]
As a linux and chruby user, I find this incomplete :trollface: 

On a more serious note, maybe it would be cool to do two things regarding the ruby version:

1. Link to `./.ruby-version` to avoid accidental docs drift when it's upgraded later, i.e. `* Install the [currently used ruby version](./.ruby-version)`
2. Instead of duplicating the install docs for the various ruby version managers, maybe rather refer to the corresponding ruby toolbox category, i.e. continuing the previous sentence `, preferrably using [a Ruby version manager like chruby, rbenv, or rvm](https://www.ruby-toolbox.com/categories/ruby_version_management)`

Regarding the other dependencies:

* As a linux user, I'd love to have some steps for that too. Postgres provides yum and apt repos for redhat/debian based distros, maybe a link to those would be cool, i.e. `Linux: Use the official package postgres repositories for [Apt](https://wiki.postgresql.org/wiki/Apt) or [Yum](https://yum.postgresql.org/)`. I also thought about linking the official postgres install docs, but the wiki page at https://wiki.postgresql.org/wiki/Detailed_installation_guides does not seem very up to date or helpful to me, so probably not such a great idea
* Redis is also required, could you please provide a section on that? As an ubuntu user, I usually use [Chris Lea's PPA for redis](https://launchpad.net/~chris-lea/+archive/ubuntu/redis-server), which is very well maintained

Maybe also provide a link to yarn's official install docs via https://yarnpkg.com/lang/en/docs/install/ and to homebrew in case folks don't know what it is via https://brew.sh/ 

Thanks!

**Comment by sacrebleu** [comment]
Good points from both of you

@colszowka I've done a wide-ranging upgrade and tested most of these commands on EC2 instances running AWS Linux or Debian.  Amusingly enough ruby-toolbox starts without REDIS, which I guess is probably used for asset and fragment caching?  I haven't dug far enough into the code to find where it is.

Anyway, thanks for the commentary.  It's already much better than it was.

**Comment by BookOfGreg** [comment]
If useful, a trivial docker-compose for development looks something like this:

```yml
version: '3'

services:
  redis:
    image: redis
    ports:
      - '6379:6379'
  db:
    image: postgres
    ports:
      - '5432:5432'
    environment:
          POSTGRES_PASSWORD: example
```

With a tweak to the database.yml like this:
```yml
default: &default
  adapter: postgresql
  encoding: unicode
  # For details on connection pooling, see Rails configuration guide
  # http://guides.rubyonrails.org/configuring.html#database-pooling
  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
  username: postgres
  password: example
  host: localhost
  port: 5432
```

**Comment by colszowka** [comment]
@BookOfGreg I created a followup issue for the docker compose env in #151, please have a look if you like :)

I'll merge this and add some further things in a followup PR. Thanks for your contribution!

**Pull request merged**